### ISSUE TITLE
Using root is required everywhere.

### ISSUE THREAD

**Issue by jtagcat** [overnode-org/overnode#12]
..at least according to the docs.

It would nice to see something like the docker group (allowing non-root users to use docker).

Any ideas, objections, agreement?

**Comment by avkonst** [comment]
Agree. Is weave can work with docker group, it should be possible. I will come back to fix it later. Now the focus is on the regression tests
